OpenEdXMobile/res/drawable/activated_item_selector.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@color/neutralBase" android:state_activated="true" />
```

Comment:
It'll be great if you can add a pressed state to give a nice ripple effect on tapping of the rows in this file and the other one as well:
```xml
<item android:drawable="@drawable/selectable_box_overlay" android:state_pressed="true" />
```